hack/.structured_logging

```diff
@@ -0,0 +1,2 @@
+pkg/kubelet/dockershim
```

Comment:
Usually this file tracks all of the failing packages that _should_ be migrated / fixed but are not yet, so we can reduce the exception list over time and eventually not allow exceptions.

I don't consider this a blocker, but just a suggestion to bring this inline with `hack/verify-*`, so far I think it has worked well. It also discourages brand new packages from failing a linter because then you also have to add them to the known failures file in that PR and someone has to approve that instead of having it fixed up front (like the shellcheck errors in an earlier iteration of this PR).

Reply:
I saw how other scripts are working, but currently all k/k packages are not migrated to structured logging. It will fail for every package, the idea here is when a package is completely migrated we add to this file and there is currently one package.

Reply:
so should we add a list of all packages here as failures?

Reply:
I think this should be an allow-list/list of things that are migrated for now, because the goal is to set this file to pkg/kubelet and cmd/kubelet by 1.21 release freeze. We are doing a slow migration and want to prevent regressions, and because structured logging is still in beta, we don't want to prevent new files from regressing quite yet. When we're ready to make the GA switch, I think we can flip this.

Reply:
+1 re: https://github.com/kubernetes/kubernetes/pull/99090#discussion_r587620282 (@ehashman comment just above this one)